tox.ini

```diff
@@ -19,19 +19,17 @@ whitelist_externals =
     mkdir
     rm
 deps =
-    django31: Django>=3.1,<3.2
-    django32: Django>=3.2,<4.0
```

Comment:
We only need to remove `django31: Django>=3.1,<3.2`.

`django32` shouldn't be removed.

Am I right?